modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -54,4 +54,14 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`spec.storage.managementState`
+
+|The Image Registry Operator sets the `spec.storage.managementState` parameter to `Managed` on new installations or upgrades of installer-provisioned infrastructure clusters on AWS or Azure.
```

Comment:
Maybe `... on new installations or upgrades of clusters using installer-provisioned infrastructure on AWS or Azure.`